Billmike/More-Recipes
# Setup API for user signup

##  What does this PR do?
Creates the API functionality for user signup.

##  Description of task to be completed
Have the following API endpoint working
-  POST ```[::1]:8080/api/v1/users/signup```

How should this be tested?
After cloninig repo, cd into morer-recipes folder and run ```npm install``` to install local dependencies
-  run ```npm start:dev``` to start local server. Make sure yu have nodemon installed
- Open up Postman and test the above route

##  Screenshot
![siggnup](https://user-images.githubusercontent.com/24798364/32137032-6fb9d186-bc10-11e7-8883-cafd10f855bf.PNG)
